package.json

```diff
@@ -92,7 +92,7 @@
     "watch-server": "Run HTTP server and restart when changes are made."
   },
   "engines": {
-    "node": ">=12.13.00"
+    "node": ">=12.13.00 || >=18.12.00"
```

Comment:
This looks a bit strange to me. Personally, I would be OK with this being just `>=18` or similar.

Reply:
Works for me, I thought it was a bit odd when I wrote it...